Stellar-reviewable-request-sale.x

```diff
@@ -5,39 +5,56 @@
 
 namespace stellar
 {
-
+//: SaleCreationRequestQuoteAsset is a structure used to provide asset code - price pairs to SaleCreationRequest
 struct SaleCreationRequestQuoteAsset {
-	AssetCode quoteAsset; // asset in which participation will be accepted
-	uint64 price; // price for 1 baseAsset in terms of quote asset
-	union switch (LedgerVersion v)
+    //: AssetCode of quote asset 
+    AssetCode quoteAsset; // asset in which participation will be accepted
+    //: Price of sale base asset in terms of quote
+    uint64 price; // price for 1 baseAsset in terms of quote asset
+    //: Reserved for future use
+    union switch (LedgerVersion v)
     {
     case EMPTY_VERSION:
         void;
     }
     ext;
 };
-
+//: SaleCreationRequest is used to create sale with provided parameters
 struct SaleCreationRequest
-{
+{   
+    //: Type of the sale
+    //: 1: basic sale
+    //: 2: crowdfunding sale
+    //: 3: fixed price sale
     uint64 saleType;
-	AssetCode baseAsset; // asset for which sale will be performed
-	AssetCode defaultQuoteAsset; // asset for soft and hard cap
-	uint64 startTime; // start time of the sale
-	uint64 endTime; // close time of the sale
-	uint64 softCap; // minimum amount of quote asset to be received at which sale will be considered a successful
-	uint64 hardCap; // max amount of quote asset to be received
+    //: Asset code of asset to sell on sale
+    AssetCode baseAsset; // asset for which sale will be performed
+    //: Asset code of the asset used to calculcate soft cap and hard cap
+    AssetCode defaultQuoteAsset; // asset for soft and hard cap
+    //: Time when sale should start
+    uint64 startTime; // start time of the sale
+    //: Time when sale should end
+    uint64 endTime; // close time of the sale
+    //: Minimal amount in default quote asset to be acquired on sale for it to be considered successful
```

Comment:
//: Minimal amount (in default quote asset) that has to be sold on sale for it to be considered successful